documentation/en/curio-pdp.md

```diff
@@ -0,0 +1,105 @@
+---
+description: This page explains how to setup a Curio PDP SP
+---
+
+# Curio PDP
+
+This guide is focused on running a PDP-only SP.  Most of the same instructions will apply for running a curio node providing both PoRep and PDP storage though you will likely need to modify the configuration defaults specified here to keep the sealing pipeline running smoothly.
+
+## Setup curio 
+
+The first step to running a curio PDP SP is to install and setup curio.
```

Comment:
Let's link the page here to installation and setup. With detailing where departure is from normal PoRep SP setup.